skins/oasis/css/core/ads.scss

```diff
@@ -270,16 +272,88 @@ body >a >img[src$="noad.gif"] {
 
 // Recovered message
 .recovered-message {
-	background: #eee;
-	color: #000;
-	font-size: 2em;
 	text-align: center;
 	padding: 10px 0;
+	min-height: 100px;
+
+	.message {
+		background: #ffffff;
+		color: #000000;
+		padding: 10px 12px;
+		border: 2px solid #000000;
+		font-size: 1.1em;
+		line-height: 1.4em;
+		position: relative;
+
+		h3 {
+			font-size: 1.2em;
+			font-weight: bold;
+		}
+
+		strong {
+			font-weight: bold;
+		}
+
+		.action-accept {
+			cursor: pointer;
+		}
+
+		.dialog-pointer {
+			width: 15px;
+			height: 15px;
+			border-top: 2px solid #000000;
+			border-left: 2px solid #000000;
+			position: absolute;
+			background: #ffffff;
+			top: -10px;
+			left: 110px;
+			-ms-transform: rotate(45deg);
```

Comment:
You can use [transform mixin](https://github.com/Wikia/app/blob/dev/skins/shared/mixins/transform.scss) instead of all those vendor prefixed properites :smiley_cat: